model/project.go

```diff
@@ -415,6 +418,16 @@ type BuildVariant struct {
 	DisplayTasks []patch.DisplayTask    `yaml:"display_tasks,omitempty" bson:"display_tasks,omitempty"`
 }
 
+// CheckRun is used to provide extra information about a github check run.
+type CheckRun struct {
+	// PathToOutputs is a local file path to an output json file for the checkrun
+	PathToOutputs *string `yaml:"path_to_outputs" bson:"path_to_outputs"`
```

Comment:
Because an empty string is a valid input for this field, it may be worth differentiating users not specifying anything vs specifying an empty string. 

Maybe the comment can mention that

Reply:
I don't think it needs to be a pointer. I added an explanation with updates in the PR description. thanks!